gradle-multi-module/jmp-cloud-bank-impl/src/main/java/module-info.java

```diff
@@ -0,0 +1,5 @@
+module jmp.bank.impl {
```

Comment:
We could export implementation via `provides` keyword\

---

24. Add to module-info jar for each implementation module :
- provides [interface] with [implementation]

Reply:
Fixed